woocommerce/woocommerce
# Update Playwright to 1.38

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates Playwright from 1.37.1 to 1.38 (the current version)

Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure all tests pass on CI
2. Pull branch, run `pnpm env:restart`
3. You will likely need to run `pnpm exec playwright install` to install the updated Playwright browsers 
4. run `pnpm test:e2e-pw` and ensure that all tests pass. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
